csharp/src/Ice/OpaqueEndpointI.cs

```diff
@@ -61,7 +61,8 @@ public override void StreamWrite(Ice.OutputStream s)
         public override string ToString()
         {
             string val = System.Convert.ToBase64String(_rawBytes);
-            return "opaque -t " + _type + " -e " + _rawEncoding.ToString() + " -v " + val;
+            short typeNum = (short)_type;
+            return $"opaque -t {typeNum.ToString(CultureInfo.InvariantCulture)} -e {_rawEncoding} -v {val}";
```

Comment:
If integer formatting is culture dependent, we need to fix other ToString methods.

Reply:
Without `InvariantCulture` you can end up with decimal separators in the string.